metrics/web/test/auth/domain/usercases/google_sign_in_usecase_test.dart

```diff
@@ -9,6 +11,23 @@ import '../../../test_utils/user_repository_mock.dart';
 void main() {
   group("GoogleSignInUseCase", () {
```

Comment:
Should we test that the user case throws when the `validateEmail` or `getGoogleSignInCredentials` method throws? Also, should we check that if the email domain is not valid, the use case does not call the `signInWithGoogle` method?